runtime.go

```diff
@@ -607,33 +579,23 @@ func getDockerVersion() (string, error) {
 
 }
 
-func getWorkingDir(basedir string, codeuri string, checkWorkingDirExist bool) (string, error) {
-
-	// Determine which directory to mount into the runtime container.
-	// If no CodeUri is specified for this function, then use the same
-	// directory as the SAM template (basedir), otherwise mount the
-	// directory specified in the CodeUri property.
-	abs, err := filepath.Abs(basedir)
-	if err != nil {
-		return "", err
-	}
-
-	dir := filepath.Join(abs, codeuri)
-
-	if checkWorkingDirExist {
+func getWorkingDir(dir string) string {
 
-		// ...but only if it actually exists
-		if _, err := os.Stat(dir); err != nil {
-			// It doesn't, so just use the directory of the SAM template
-			// which might have been passed as a relative directory
-			dir = abs
+	// If the template filepath isn't set, just use the cwd
+	if dir == "" {
+		cwd, err := os.Getwd()
```

Comment:
There are three cases:
1. CWD is given as input parameter to CLI
2. CodeUri specifies the directory or file
3. Nothing works, then fall back to os.Getwd()

I believe point 2 is not handled